client/my-sites/subscriptions/index.tsx

```diff
@@ -10,13 +10,15 @@ import { makeLayout, render } from 'calypso/controller';
 
 const SitesView = () => <span>Sites View</span>;
 const CommentsView = () => <span>Comments View</span>;
-const SettingsView = () => <SubscriptionManager.UserSettings />;
 
```

Comment:
```suggestion
const SettingsView = () => {
    const { data: settings } = Reader.useSubscriptionManagerUserSettingsQuery();
    return <SubscriptionManager.UserSettings value={ settings } />;
}

```
However, if we will be able to use the exact same query and mutation hook for querying and mutating the data for a logged in user, then we may implement it within the `UserSettings` component, as suggested by @TimBroddin.

Reply:
Yes, it's exactly the same :)

Reply:
Thanks for the feedback Tim & Maciej! I will applied the suggested changes.

Reply:
Addressed in the following commit: https://github.com/Automattic/wp-calypso/pull/74763/commits/aebfe81d3dc02b349251c79200e2554ddb91eb5e.